.github/workflows/archive/talawa_mobile_md_mdx_format_adjuster.py

```
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
```

Ruff (0.11.9)

Shebang is present but file is not executable

(EXE001)